options/locale/locale_en-US.ini

```diff
@@ -352,6 +352,7 @@ enable_update_checker = Enable Update Checker
 enable_update_checker_helper = Checks for new version releases periodically by connecting to gitea.io.
 env_config_keys = Environment Configuration
 env_config_keys_prompt = The following environment variables will also be applied to your configuration file:
+config_write_file_prompt = These configuration options will be written into:
```

Comment:
To make it translatable ,it should be `These .. into: %s`

Reply:
https://github.com/go-gitea/gitea/blob/cd598bfd3d997f89700a01d84434a4d09ca8d5e9/templates/install.tmpl#L341-L342  

I'm not sure if I understand correctly; The file path has already been changed into an element node, so it's not suitable to be passed as a placeholder to `config_write_file_prompt`, right?

Reply:
```
{{$html := HTMLFormat `<label>%s</label>` .CustomConfFile}}
{{ctx.Locale.Tr `tip` $html}}
```

Reply:
It's not that it can't be done, but I'm just wondering, is it necessary to make it this complicated?

Reply:
If you were doing i18n, then it needs to make it translatable for all languages, many languages do not use "A: b" syntax.

Otherwise, keep it in English if it doesn't matter (that's why I didn't translate it)